en-2016-07-17-latex-template-for-responses-to-referees-reports/statement.tex

```diff
@@ -1,63 +1,64 @@
-%
 % Title:    A LaTeX Template For Responses To a Referees' Reports
 % Author:   Petr Zemek <s3rvac@gmail.com>
 % Homepage: https://blog.petrzemek.net/2016/07/17/latex-template-for-responses-to-referees-reports/
 % License:  CC BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
-%
-\documentclass[a4paper,10pt]{article}
+\documentclass[10pt]{article}
 
-% Packages
+% Allow Unicode input (alternatively, you can use XeLaTeX or LuaLaTeX)
 \usepackage[utf8]{inputenc}
 
-% View
-\setlength{\hoffset}{-1.5cm}
-\setlength{\voffset}{-1.5cm}
-\setlength{\textheight}{23.0cm}
-\setlength{\textwidth}{15.2cm}
-
-% Paragraph formatting (no indentation)
-\setlength{\parskip}{1.3ex plus 0.2ex minus 0.2ex}
-\setlength{\parindent}{0pt}
-
-% Environments
-\newenvironment{comment}
-	{\begin{tabular}{|p{\textwidth}}\begin{tt}}
-	{\end{tt}\end{tabular}}
-\newenvironment{response}
-	{\vspace{0.2cm}}
-	{\vspace{0.2cm}}
-
-% Title
-\title{Statement on the Revision of $\langle$Paper ID$\rangle$ \\ Based on the Referees' Report}
-\author{Author1 and Author2 and Author3}
-\date{\today}
+\usepackage{microtype,xparse,tcolorbox}
+\newenvironment{reviewer-comment }{}{}
+\tcbuselibrary{skins}
+\tcolorboxenvironment{reviewer-comment }{empty,
+  left = 1em, top = 1ex, bottom = 1ex,
+  borderline west = {2pt} {0pt} {black!20},
+}
+\ExplSyntaxOn
+\NewDocumentEnvironment {response} { +m O{black!20} } {
```

Comment:
The `+m` means 'one long mandatory argument' (where *long* is TeX lingo for 'this can be multiple paragraphs').

The `O{}` is an optional argument with a default value.

See `texdoc xparse` for more details.